rootfs-builder/rootfs.sh

```diff
@@ -147,7 +147,7 @@ copy_kernel_modules()
 
 OSBUILDER_VERSION="unknown"
 
-while getopts c:ho:r: opt
+while getopts a:ho:r: opt
```

Comment:
Good catch. Please can you create a separate commit for this though.

Note that you don't a "Fixes #XXX" comment for this new commit - as long as there is atleast one "Fixes #XXX" comment in the commit range, our CI checks will be happy :smile:

Reply:
Sorry. I will create new issue and commit for this and follow the CI-checking format~😁

Reply:
You do **not** need to create a new issue for this change! :smile:

Reply:
No - when you raise a PR, it will have atleast one commit. But only one of those commits needs to have the "Fixes #XXX" comment in it.

So all you need to do is create a new commit for the `getopts` change.  The rest of the requirements in https://github.com/kata-containers/community/blob/master/CONTRIBUTING.md#patch-format still apply though.

So for example your new commit could look something like this:

```
rootfs: Fix incorrect getopts call

The `getopts` call in the rootfs builder was incorrect
meaning the `-a $agent_version` option would never have worked.

Signed-off-by: Penny Zheng penny.zheng@arm.com
```

Reply:
Hi @Pennyzct - I see where your confusion came from now. I've raised https://github.com/kata-containers/community/pull/29 to make the wording clearer on this point! :smile:

Reply:
LoL😄, so grateful for the detailed guideline, i'm really a rookie in github.😥